src/components/TeeTimeContainer/TeeTimeContainer.css

```diff
@@ -45,6 +46,10 @@
     border-top-right-radius: 16px;
   }
 
+  .container-title > h2 {
+    margin-bottom: unset;
```

Comment:
Is unset the same as setting it to 0?

Reply:
Essentially yeah, it reverts it back to its original value. 

I did this for these because I think there is a style that was added in EventForm that sets margins on all <h2> and <button> elements, so we might want to fix that at some point.